panel/chat/feed.py

```diff
@@ -681,6 +683,55 @@ def stream(
         self.param.trigger("_post_hook_trigger")
         return message
 
+    def create_steps(
```

Comment:
Maybe this should simply be done by doing `.stream("", user='foo', steps=True)`? The `steps` option might need a better name.

Reply:
See https://github.com/holoviz/panel/pull/6617#discussion_r1625104395